src/model/encoding/convertFromHTMLToContentBlocks.js

```diff
@@ -132,11 +132,17 @@ function getListBlockType(
 function getBlockMapSupportedTags(
   blockRenderMap: DraftBlockRenderMap
 ): Array<string> {
+  // Some blocks must be treated as unstyled when not present on the blockRenderMap
   const unstyledElement = blockRenderMap.get('unstyled').element;
-  return blockRenderMap
+  const defaultUnstyledSet = new Immutable.Set(['p']);
```

Comment:
hi @howardjing please feel free to pick this up, i am unable to contribute to Draft until the 14th of October when i will resume work in here :)